nimble/host/audio/include/audio/ble_audio.h

```diff
@@ -562,6 +583,66 @@ struct ble_audio_event_codec_unregistered {
     const struct ble_audio_codec_record *record;
 };
 
+/** @brief BASS Source Removed */
+struct ble_audio_event_bass_remote_scan {
+    /** Connection Handle of Broadcast Assistant that is performing Scan procedure for us */
+    uint16_t conn_handle;
+};
+
+/** @brief BASS Source Added */
+struct ble_audio_event_bass_source_added {
+    /** Source ID */
+    uint8_t source_id;
+
+    /** Event status. 0 on success, BLE_HS Error code otherwise */
+    uint8_t status;
+};
+
+/** @brief BASS Source Modify */
+struct ble_audio_event_bass_source_modify {
+    /** Source ID */
+    uint8_t source_id;
+
+    /** Event status. 0 on success, BLE_HS Error code otherwise */
+    uint8_t status;
+
+    /** PA Sync operation to be performed by Broadcast Sink */
+    enum {
+        /** Do not synchronise to PA */
+        BLE_AUDIO_EVENT_BASS_NO_SYNC,
+
+        /** Synchronise to PA without using PAST */
+        BLE_AUDIO_EVENT_BASS_SYNC,
+
+        /** Terminate PA Sync */
+        BLE_AUDIO_EVENT_BASS_SYNC_TERMINATE,
+
+        /** Synchronise to PA using PAST */
+        BLE_AUDIO_EVENT_BASS_SYNC_PAST,
+    } pa_sync_operation;
+};
+
+/** @brief BASS Set Broadcast Code */
+struct ble_audio_event_bass_set_broadcast_code {
+    /** Source ID */
+    uint8_t source_id;
+
+    /** Event status. 0 on success, BLE_HS Error code otherwise */
+    uint8_t status;
+
+    /** Source ID */
+    uint8_t broadcast_code[BLE_AUDIO_BROADCAST_CODE_SIZE];
+};
```

Comment:
What's the purpose of having `status` needed in this event?

Reply:
Yes, that's what I think of. There is no point of informing app that peer performed invalid operation.